src/node/rpc/frontend.h

```diff
@@ -22,6 +22,17 @@
 
 namespace ccf
 {
+  struct RequestArgs
+  {
+    enclave::RPCContext& rpc_ctx;
+    Store::Tx& tx;
+    CallerId caller_id;
+    const std::string& method;
+    const nlohmann::json& params;
+    const SignedReq& signed_request;
+  };
+
+  template <typename CT = void>
```

Comment:
I would prefer this to be called `CallerTable` instead of `CT`

Reply:
As discussed, we will go for `CT` for now.